x-pack/plugins/transform/public/app/sections/create_transform/components/step_define/step_define_summary.tsx

```diff
@@ -66,84 +59,76 @@ export const StepDefineSummary: FC<Props> = ({
-                  >
-                    <span>{searchString}</span>
-                  </EuiFormRow>
-                )}
-                {typeof searchString === 'undefined' &&
-                  !isDefaultQuery(pivotQuery) &&
-                  !isMatchAllQuery(pivotQuery) && (
-                    <EuiFormRow
-                      label={i18n.translate(
-                        'xpack.transform.stepDefineSummary.queryCodeBlockLabel',
-                        {
-                          defaultMessage: 'Query',
-                        }
-                      )}
-                    >
-                      <EuiCodeBlock
-                        language="js"
-                        fontSize="s"
-                        paddingSize="s"
-                        color="light"
-                        overflowHeight={300}
-                        isCopyable
-                      >
-                        {JSON.stringify(pivotQuery, null, 2)}
-                      </EuiCodeBlock>
-                    </EuiFormRow>
-                  )}
-              </Fragment>
-            )}
-
-            {searchItems.savedSearch !== undefined && searchItems.savedSearch.id !== undefined && (
+    <div data-test-subj="transformStepDefineSummary">
+      <EuiForm>
+        {searchItems.savedSearch === undefined && (
+          <Fragment>
+            <EuiFormRow
+              label={i18n.translate('xpack.transform.stepDefineSummary.indexPatternLabel', {
+                defaultMessage: 'Index pattern',
+              })}
+            >
+              <span>{searchItems.indexPattern.title}</span>
+            </EuiFormRow>
+            {typeof searchString === 'string' && (
               <EuiFormRow
-                label={i18n.translate('xpack.transform.stepDefineSummary.savedSearchLabel', {
-                  defaultMessage: 'Saved search',
+                label={i18n.translate('xpack.transform.stepDefineSummary.queryLabel', {
+                  defaultMessage: 'Query',
                 })}
               >
-                <span>{searchItems.savedSearch.title}</span>
+                <span>{searchString}</span>
               </EuiFormRow>
             )}
-
-            <EuiFormRow
-              label={i18n.translate('xpack.transform.stepDefineSummary.groupByLabel', {
-                defaultMessage: 'Group by',
-              })}
-            >
-              <GroupByListSummary list={groupByList} />
-            </EuiFormRow>
-
-            <EuiFormRow
-              label={i18n.translate('xpack.transform.stepDefineSummary.aggregationsLabel', {
-                defaultMessage: 'Aggregations',
-              })}
-            >
-              <AggListSummary list={aggList} />
-            </EuiFormRow>
-          </EuiForm>
-        </div>
-      </EuiFlexItem>
-
-      <EuiFlexItem>
+            {typeof searchString === 'undefined' &&
+              !isDefaultQuery(pivotQuery) &&
+              !isMatchAllQuery(pivotQuery) && (
```

Comment:
nit: defined the variable before return statement
```suggestion
        const isModifiedQuery =    {typeof searchString === 'undefined' &&
              !isDefaultQuery(pivotQuery) &&
              !isMatchAllQuery(pivotQuery);
```

Reply:
Done in bf06d9e.